Marlin/Configuration_adv.h

```diff
@@ -2743,7 +2599,7 @@
  * TMCStepper library is required to use TMC stepper drivers.
  * https://github.com/teemuatlut/TMCStepper
```

Comment:
Maybe move this URL/section here (and use Marlin's repo since Trinamic nuked theirs):

```
 * The TMC26XStepper library is required for TMC26X stepper drivers.
 * https://github.com/MarlinFirmware/TMC26XStepper
 ```

Reply:
Done.